dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,30 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO010.001 Attempt to enroll expired certificate
```

Comment:
@WiktorG351 please, split it into two tests, one that attempt to enroll expired  cert, which should fail in case where we are able to enroll the certificate

Second that try to boot image signed with expired cert (assuming the cert was enrolled before end of date). This should also fail if the image boots.

Reply:
After thinking about it, second test would repeat things the first one does. So lets have one test here.

Reply:
I rebased it and make small changes. Merging.